packages/wonder-blocks-modal/src/themes/default.ts

```diff
@@ -1,36 +1,69 @@
-import * as tokens from "@khanacademy/wonder-blocks-tokens";
+import {
+    border,
+    semanticColor,
+    spacing,
+} from "@khanacademy/wonder-blocks-tokens";
 
 const theme = {
-    color: {
-        bg: {
-            inverse: tokens.color.darkBlue,
+    /**
+     * Shared tokens
+     */
+    root: {
+        // TODO(WB-1852): Remove light variant.
+        color: {
+            inverse: {
+                background: semanticColor.surface.inverse,
+                foreground: semanticColor.text.inverse,
+            },
         },
-        text: {
-            inverse: tokens.color.white,
-            secondary: tokens.color.offBlack64,
+        border: {
+            radius: border.radius.medium_4,
+            width: border.width.thin,
         },
-        shadow: {
-            default: tokens.color.offBlack16,
+    },
+    /**
+     * Building blocks
+     */
+    backdrop: {
+        color: {
```

Comment:
For the theme component tokens with building blocks, do you think it's helpful to keep the additional separation between `color`,`spacing`, etc? Curious what you think!